packages/std-contracts/src/test/Sample.t.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
```

Comment:
Since this test is mostly testing that a system's `SystemStorage` is behaving correctly (and the system can be executed), I'd prefer moving the test into a `System.t.sol` in `solecs/src/test`. All it has to test is that after initializing the system, `SystemStorage.world()/components()` corresponds to `world/world.components()`.